packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -1114,7 +1114,7 @@ class SlideshowView extends React.Component {
       GALLERY_CONSTS[optionsMap.behaviourParams.gallery.layoutDirection].RIGHT_TO_LEFT;
     const { hideLeftArrow, hideRightArrow } = this.state;
 
-    const isScrollStart = this.isScrollStart();
+    const isScrollStart = this.isScrollStart(props);
     const isFirstItem = this.isFirstItem();
```

Comment:
Maybe cover also this function?

Reply:
We're checking if `this.state.activeIndex === 0`, no props needed.